vendor/modules.txt

```diff
@@ -422,3 +486,163 @@ gotest.tools/v3/internal/difflib
 gotest.tools/v3/internal/format
 gotest.tools/v3/internal/source
 gotest.tools/v3/poll
+# k8s.io/api v0.19.0
```

Comment:
This is too much. What needs it and how can we get rid of it?

Reply:
This is for CRI compatibility (secret-based registry auth: [doc](https://github.com/containerd/stargz-snapshotter/blob/master/docs/overview.md#authentication)). I think we can get rid of this in buildkit. I'll work on this.